js/interface.js

```diff
@@ -478,11 +478,13 @@ function browseDataSource(id) {
   });
 }
 
-function createDataSource(createOptions) {
+function createDataSource(createOptions, options) {
   createOptions = createOptions || {};
+  options = options || {};
 
   return Fliplet.Modal.prompt({
     title: 'Enter the name of your new Data Source',
+    value: _.get(options, 'value', '')
```

Comment:
Semantically, the option provided is for the data source name, not for data source value.

```suggestion
    value: _.get(options, 'name', '')
```

Reply:
done